### PR TITLE
Add widget to get the current number of confirmed exoplanets

### PR DESCRIPTION
**Pull request by timothygebhard**
Initial commit of the widget that retrieves the current number of confirmed of exoplanets from the NASA Exoplanet Archive (using `astroquery`), and groups the numbers by different categories (i.e., the detection method and the approximate class of the planet).

### CODE REVIEW

File: stateoftheuniverse/widgets/exoplanet_count/exoplanet_count.py [hunk 1]
```diff
@@ -0,0 +1,127 @@
...
+
+    Returns:
+        The approximate planet class, which is one of the following:
+            ["Asteroidan", "Mercurian", "Subterran", "Terran",
+             "Superterran", "Neptunian", "Jovian", "N/A", "Other"]
+    """
+
+    # Convert from Jupiter units to Earth units, if necessary
+    if units == 'Jupiter':
+        mass *= 317.907
```
Comment (ojhall94) [inline stateoftheuniverse/widgets/exoplanet_count/exoplanet_count.py]: For consistency we should use the `astropy.units` module for these conversions. I can help with this if you need.

**Review by ojhall94 — COMMENTED** [review]
This is awesome code, great docstrings!! I'll test it on my machine later :)

File: stateoftheuniverse/widgets/exoplanet_count/exoplanet_count.py (hunk 1 above)
Comment (ojhall94) [reply stateoftheuniverse/widgets/exoplanet_count/exoplanet_count.py]: Actually starting to think that all the stuff that relies on units here should be astropy unit compatible cos that would be super neat :D

Comment (timothygebhard) [reply stateoftheuniverse/widgets/exoplanet_count/exoplanet_count.py]: Sounds like a good idea! I guess we should all briefly coordinate about this, though, to make sure it's implemented consistently across all widgets?

Comment (ojhall94) [reply stateoftheuniverse/widgets/exoplanet_count/exoplanet_count.py]: We decided that if something has a unit, it should have an astropy unit.

**Comment by timothygebhard** [comment]
I changed the code so that all numbers that have a unit are now of type `astropy.units.Quantity`, which actually simplifies some things 🙂

**Review by ojhall94 — APPROVED** [review]
Looks good to me! We'll unit test at a later date.

**Pull request merged**